src/tools/cmod/3dstocmod/3dstocmod.cpp

```diff
@@ -15,11 +15,13 @@
 
 #include <cel3ds/3dsread.h>
 #include <celmath/mathlib.h>
+#include <celutil/logger.h>
 
 #include "cmodops.h"
 #include "convert3ds.h"
 #include "pathmanager.h"
 
+using namespace celestia::util;
```

Comment:
I think it would be better to just import `GetLogger` rather than the whole namespace here, this is similar to how this is imported in the main part of the codebase.

```C++
using celestia::util::GetLogger;
```

Reply:
True. Fixed.